.circleci/config.yml

```diff
@@ -23,6 +23,9 @@ workflows:
             parameters:
               configuration: ["Debug", "Release"]
               report_failure: [true]
+      - build-debug-app:
+          report_failure: true
+          xcode: "12.4.0"
```

Comment:
our min version of Xcode is 12.2, should we switch to that? or maybe add matrix

Reply:
We should definitely consider setting up matrixes for supported Xcode versions. I'll create a separate ticket for that though, because I think it deserves a well-thought-out plan and discussion with the team.